### PR TITLE
Update Plugin Docs for up to date topic information

### PR DESCRIPTION
**Pull request by therishidesai**

### CODE REVIEW

**Review by e3m3 — DISMISSED** [review]

**Comment by charmoniumQ** [comment]
We should separate `subscribed` into `subscribed/scheduled` and `asynchronously read`.

**Review by charmoniumQ — DISMISSED** [review]

**Review by e3m3 — DISMISSED** [review]

**Review by mhuzai — CHANGES_REQUESTED** [review]
Please do not merge without addressing these comments first.

File: docs/illixr_plugins.md
Comment (e3m3) [reply docs/illixr_plugins.md:77]: Does this also include the `mtp_logger` in `timewarp_gl/plugin.cpp`?
```cpp
        , _m_mtp{sb->publish<std::chrono::duration<double, std::nano>>("mtp")}
        , _m_frame_age{sb->publish<std::chrono::duration<double, std::nano>>("warp_frame_age")}
        , _m_offload_data{sb->publish<texture_pose>("texture_pose")}
        , timewarp_gpu_logger{record_logger_}
        , mtp_logger{record_logger_}
```

File: docs/illixr_plugins.md
Comment (e3m3) [reply docs/illixr_plugins.md:85]: Should `imu_raw` be called `fast_pose` here?
```cpp
   -   Asynchronously *reads* `imu_raw` on `imu_raw` topic.
```
```cpp
    debugview(std::string name_, phonebook *pb_)
        : threadloop{name_, pb_}
        , sb{pb->lookup_impl<switchboard>()}
        , pp{pb->lookup_impl<pose_prediction>()}
        , _m_slow_pose{sb->subscribe_latest<pose_type>("slow_pose")}
        , _m_fast_pose{sb->subscribe_latest<imu_raw_type>("imu_raw")}
        //, glfw_context{pb->lookup_impl<global_config>()->glfw_context}
    {}
```

File: docs/illixr_plugins.md
Comment (mhuzai) [reply docs/illixr_plugins.md:77]: No, no, the loggers stay. They go to sqlite, which we do need. It's only `_m_mtp` and `_m_frame_age` that need to go -- they are unused.

File: docs/illixr_plugins.md
Comment (e3m3) [reply docs/illixr_plugins.md:104]: Yup the name should remain `dataflow.dot.png`. I think the images are rendered assuming that they are in `docs/images`, so I will move the `.png` there so that we don't need the `..`.

Comment (e3m3) [reply docs/illixr_plugins.md:104]: > Yup the name should remain `dataflow.dot.png`. I think the images are rendered assuming that they are in `docs/images`, so I will move the `.png` there so that we don't need the `..`.

This doesn't quite work. The images on a page are assumed to be in a directory with the same name as the markdown. After moving to `docs/images/dataflow.dot.png`, we need to specify the path as `../images/dataflow.dot.png`.

File: docs/illixr_plugins.md
Comment (mhuzai) [reply docs/illixr_plugins.md:85]: `fast_pose` could work, but the problem is that `imu_raw` also includes the last two (biased) IMU readings, so it's not just the pose. Open to naming suggestions.

Comment (mhuzai) [reply docs/illixr_plugins.md:88]: Right. Pose prediction uses all of `imu_raw` but debugview only uses the pose part of it, and we don't anticipate using the biases in debugview. So we can point that out. But I'm still not sure whether we should call the pose part `fast_pose` because it is not exactly a pose type, but rather has the position and quaternion stored separately.

File: docs/illixr_plugins.md
Comment (mhuzai) [reply docs/illixr_plugins.md:80]: Good to go once the above are removed from the doc.

**Review by mhuzai — APPROVED** [review]

**Review by charmoniumQ — APPROVED** [review]

**Pull request merged**
